### DISCUSSION TITLE
Didn't charge at all - No house load data available

### DISCUSSION THREAD

**Discussion by C00K50N** [nathanmarlor/foxess_em#137] [General]
No charge attempted at all last night so I've woken up to a battery at 11%, same as it was last night.  Looking at the logs, it seems no house load data was available at 01:40 when it's scheduled to check and start charging.

Any ideas what might have happened?

And can I request again that if possible, could the integration be tweaked so that at the end of the cheap period we turn the Fox Cloud force charge switches back on, for the full time specified as eco period?  That way if this integration doesn't trigger a charge there's a chance it would fall back on doing a full charge without the integration.  

Better too full a charge at cheap rate than not enough.

**Comment by nathanmarlor** [comment]
No house load would indicate an issue with HA history service. Any logs for the eco start period? 

Things have been busy for me the last few weeks so haven't had chance to work on the integration. Setting the force charge switches to default on is next on my to-do list though.

**Reply by C00K50N** [reply]
Thanks for the quick reply!  And no worries, it's a great integration already.  I really appreciate all the work you've put into it.

Logs only seem to start at 01:39 - does that mean my HA did a restart?!  As far as I know we didn't have any power drop, and internet connection seems to have been rock solid.  Currently looking to see if something is causing HA to auto restart but I'm struggling...

**Comment by nathanmarlor** [comment]
Charge is now on by default as of 1.6.0b4 -> https://github.com/nathanmarlor/foxess_em/releases/tag/v1.6.0b4

:)
